src/overlap_check/ProgressBar.hpp

```diff
@@ -0,0 +1,24 @@
+
+#ifndef DAGMC_PROGRESSBAR_H
+#define DAGMC_PROGRESSBAR_H
+
+#include <string>
```

Comment:
`<string>` header not referenced and can be omitted.

Reply:
Moved this to the CPP file to reduce header creep